src/app/client-view/client-view.component.ts

```diff
@@ -16,11 +16,20 @@ export class ClientViewComponent implements OnInit {
   ];
   selectedView: string = 'Training';
 
+
   constructor() {}
 
   ngOnInit(): void {}
 
   onSelect(view: string): void {
     // this.selectedView = view;
   }
```

Comment:
I forgot to remove these before when I started with the routing.
```suggestion

  constructor() {}

  ngOnInit(): void {}
```